config/inits/init.rb

```diff
@@ -1,9 +1,12 @@
-$telegram_bot_token = 'your_bot_token'
-$namespace = 'project_name'
+require 'dotenv/load'
+
+
+$telegram_bot_token = ENV['BOT_TOKEN']
+$namespace = File.basename(Dir.getwd)
```

Comment:
:+1: 

tapi nama folder-nya bisa jadi ga ngikutin standard namespace sih, di-`.gsub(/\s/, '').underscore`-in aja